Kek.js

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+export class Kek {
+  /**
+   * Creates a new instance of a key encryption key.
+   *
+   * @param {String} id the ID of this key.
+   * @param {Object} signer an API for creating digital signatures using an
+   *   authentication key for a KMS service.
+   * @param {KmsService} kmsService the kmsService to use to perform key
+   *   operations.
+   * @param {String} kmsPlugin the ID of the KMS plugin to use.
+   *
+   * @return {Kek}.
+   */
+  constructor({id, signer, kmsService, kmsPlugin}) {
+    this.id = id;
+    // TODO: support other algorithms
+    this.algorithm = 'A256KW';
```

Comment:
Looks like this isn't used yet... but is the string supposed to be the same `AES-KW` used in AccountMasterKey?

Reply:
No, it's not supposed to be the same. There is some incongruity between WebCrypto API and JOSE... this is a JOSE algorithm representation of the WebCrypto algorithm `{name: 'AES-KW', length: 256}`. JOSE squeezes that info into a single string.